### PR TITLE
[SOAR-10035] SQL Plugin pymssql update

### PR DESCRIPTION
**Pull request by llaszuk-r7**
### Description

Describe the proposed changes:

  - Update pymssql version library to `2.2.5`

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

**Review by cmcnally-r7 — APPROVED** [review]

File: plugins/sql/help.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ For the SQL query action, be sure that your query is valid SQL.
 
 # Version History
 
+* 3.0.5 - Update pymssql version library
```
Comment (blaxminarayan-r7) [inline plugins/sql/help.md:122]:
```suggestion
* 3.0.5 - Update pymssql version library to support azuresql
```

File: plugins/sql/plugin.spec.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: sql
 title: SQL
 description: The SQL plugin allows you to run SQL queries
-version: 3.0.4
+version: 3.0.5
 supported_versions: ["MySQL 8.0.21", "MSSQL 2019 15.0.2000.5", "PostgresSQL 13.0"]
```
Comment (blaxminarayan-r7) [inline plugins/sql/plugin.spec.yaml:8]: Can you please add Azure SQL if we are supporting with this version

**Review by blaxminarayan-r7 — CHANGES_REQUESTED** [review]

File: plugins/sql/plugin.spec.yaml (hunk 2 above)
Comment (llaszuk-r7) [reply plugins/sql/plugin.spec.yaml:8]: AzureSQL equals MSSQL

Comment (blaxminarayan-r7) [inline plugins/sql/plugin.spec.yaml:8]: Customer was not able to connect to: ```I’ve read that Azure SQL always runs on the latest version of SQL Server, which [looks like CU16 15.0.4223.1](https://support.microsoft.com/en-us/topic/kb4518398-sql-server-2019-build-versions-782ed548-1cd8-b5c3-a566-8b4f9e20293a), whereas the R7 SQL plugin documentation claims to support MSSQL 2019 15.0.2000.5, which was the RTM version of SQL Server 2019.

I couldn’t find a definitive source, and granted the versions aren’t exactly aligned, but hopefully there shouldn’t be any compatibility issues there given that they’re both 15.0.x``` Do we support the CU16 15.0.4223.1 with the update. If so can you please add the version. Also, kindly  update the Version History

File: plugins/sql/help.md [hunk 3]
```diff
@@ -119,6 +119,7 @@ For the SQL query action, be sure that your query is valid SQL.
 
 # Version History
 
+* 3.0.5 - Update pymssql version library to support azuresql
```
Comment (blaxminarayan-r7) [inline plugins/sql/help.md:122]:
```suggestion
* 3.0.5 - Update pymssql version library to support latest MSSQL update 2019 15.0.4223.1
```

**Review by blaxminarayan-r7 — APPROVED** [review]

**Pull request merged**